.github/workflows/statutter.yml

```diff
@@ -0,0 +1,50 @@
+---
+name: Statutter CI
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install packages
+        run: |
+          sudo apt-get update -qq
+          sudo apt-get install -qq \
+            texlive texlive-font-utils \
+            texlive-lang-european \
+            texlive-latex-extra \
+            latexmk \
+            ghostscript
+      # Need pandoc 2.9.* for compatibility with pandoc-section-links
+      - name: Install pandoc
```

Comment:
You surely know this better than me, but would it not be better to use a docker container that was built with pandoc? That makes sure it works even if the links break.

Reply:
Probably yeah, but I just migrated the travis conf to this. Could look into it though

Reply:
I feel gh actions using docker containers is weird and dumb, so I would just keep this. The link will 99% not be removed. ever.